CHANGELOG.rst

```diff
@@ -19,8 +19,76 @@
   ``mutation.index`` will still work through deprecated aliases,
   but new code should use a more cite-centric approach.
 
+- The ``TreeSequence.diffs()`` method no longer works. Please use
+  the ``TreeSequence.edge_diffs()`` method instead.
 
-**Deprecated aliases**:
+- TreeSequence.get_num_records() no longer works. Any code using
+  this or the records() iterator should be recast to work with
```

Comment:
probably not an issue but: maybe s/recast/rewritten/ as casting has that other meaning in CS